src/results.jl

```diff
@@ -47,9 +46,10 @@ mutable struct Result
             _FALLBACK_CONVERSION,
         )
 
-        jl_result.column_oids = col_oids = map(1:num_columns(jl_result)) do col_num
-            libpq_c.PQftype(jl_result.result, col_num - 1)
-        end
+        jl_result.column_oids =
```

Comment:
No dangling assignments

https://github.com/invenia/BlueStyle/#whitespace

Reply:
Please apply this in other places as well. I will make an issue for including specifically chained assignments on one line in BlueStyle as well to be official.